src/rest-server/src/controllers/v2/job.js

```diff
@@ -35,7 +35,7 @@ const update = asyncHandler(async (req, res) => {
       throw createError('Conflict', 'ConflictJobError', `Job ${frameworkName} already exists.`);
     }
   } catch (error) {
-    if (error.code === 'UnknownError') {
+    if (error.code !== 'NoJobError') {
```

Comment:
Can log this?
```js
if (error.code === 'NoJobError') {
  log.Debug('...');
  return;
}

throw error;
```

Reply:
it's not an error if job not found here, no need to log.